chore: sync with R versions of shiny, bslib, htmltools (#1266)
posit-dev/py-shiny: CHANGELOG.md

* `ui.panel_conditional()` now adds the `.shiny-panel-conditional` class to the `<div>` element wrapping the conditional panel contents. (#1257)

> * Unwanted double padding is also removed for items in `ui.panel_conditional()` the parent container uses `gap` for spacing multiple elements (e.g., when used in `ui.layout_columns()`, `ui.page_fillable()`, etc). (#1266)

Reviewer: I think I missed documenting the fix for `@render.ui`.

```suggestion
* `ui.panel_conditional()` and `@render.ui` no longer result in unwanted double padding when their parent container uses `gap` for spacing multiple elements (e.g., when used in `ui.layout_columns()`, `ui.page_fillable()`, etc). (#1266)
```

Author: I think that's included already, you just mentioned `ui.output_ui()` instead of `@render.ui`

https://github.com/posit-dev/py-shiny/blob/41aa9ef19f6f6854e0661f114ed2433d44ad5714/CHANGELOG.md?plain=1#L62

I originally had basically your suggestion and then changed it when I realized that we had already shipped the `output_ui()` part of this.

Reviewer: Ahh, good catch. It'd still be nice to have the wording be a bit more consistent.

```suggestion
* `ui.panel_conditional()` no longer results in unwanted double padding when the parent container uses `gap` for spacing multiple elements (e.g., when used in `ui.layout_columns()`, `ui.page_fillable()`, etc). (#1266)
```
